CHANGELOG.md

```diff
@@ -21,19 +21,41 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ### Added
-- Sign generated SBOMs and store container image SBOMs in registry for easier usage.
 
 ### Changed
 <!-- For changes in existing functionality.  -->
+
+### Deprecated
+<!-- For soon-to-be removed features. -->
+
+### Removed
+<!-- For now removed features. -->
+
+## [2.2.0] - 2022-11-08
+
+### Added
+
+- Sign generated SBOMs and store container image SBOMs in registry for easier usage.
+- Support for Constellation on AWS.
```

Comment:
Should we put *support for AWS* first to give it more visibility?

Or in general, are there any opinions on sorting changelog items?

Reply:
Don't think we need to worry about sorting of items in the changelog document, if we want to highlight specific changes we can do so in the release notes